web/modules/custom/solution/solution.behat.inc

```diff
@@ -226,18 +204,8 @@ class SolutionSubContext extends DrupalSubContextBase implements DrupalSubContex
    * @throws \InvalidArgumentException
    *   Thrown when a solution with the given name does not exist.
    */
-  protected function getSolutionByName($name) {
-    $query = \Drupal::entityQuery('rdf_entity')
-      ->condition('rid', 'solution')
-      ->condition('label', $name)
-      ->range(0, 1);
-    $result = $query->execute();
-
-    if (empty($result)) {
-      throw new \InvalidArgumentException("The solution with the name '$name' was not found.");
-    }
-
-    return Rdf::load(reset($result));
+  public static function getSolutionByName($title) {
+    return self::getRdfEntityByLabel($title, 'solution');
```

Comment:
There should be no public static methods in this class. The only public methods should be the step definitions. There is no reason why this class should ever be loaded by external code.